src/bin/pg_autoctl/keeper_config.c

```diff
@@ -131,6 +131,7 @@
 							&(config->postgresql_restart_failure_max_retries), \
 							POSTGRESQL_FAILS_TO_START_RETRIES)
 
+
```

Comment:
Nitpicking: extra whitespace added.

Reply:
I realise the proper answer to those is to setup some automatic indenting rules on the repository, but we failed to do that up to now, so I guess we still have to be careful about it :(

Reply:
citus_indent would have been good